test/test_helper.rb

```diff
@@ -0,0 +1,18 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+require 'rails/test_help'
+
+class ActiveSupport::TestCase
```

Comment:
__[development][rubocop]__
* Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.